central/cve/telemetry/expression.go

```diff
@@ -0,0 +1,120 @@
+package telemetry
+
+import (
+	"math"
+	"strconv"
+	"strings"
+
+	"github.com/stackrox/rox/pkg/glob"
+)
+
+var ops = []string{"=", "!=", "<=", ">=", "<", ">"} // The order matters, as the
```

Comment:
I think we should point out here that the filtering logic is a performance optimization. I wonder if it is a pre-mature one - it seems to complicate the code significantly, and the metric volume could otherwise be controlled by the labels.

Not saying we should not implementing filtering - also considering it's already done now - but I would not have implemented it myself, and I don't think it provides much additional value over the label selection alone.